shlr/sdb/src/buffer.c

```diff
@@ -2,7 +2,7 @@
 
 #include "sdb/buffer.h"
 
-void buffer_init(buffer *s, BufferOp op, int fd, char *buf, ut32 len) {
+void buffer_initialize(buffer *s, BufferOp op, int fd, char *buf, ut32 len) {
```

Comment:
at some point this can cause some name conflict issues but this api shouldnt be public anyway.. but its fine to update it for now. thank you!